x-pack/test/licensing_plugin/apis/changes.ts

```diff
@@ -97,30 +97,71 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
         isEnabled: true,
       });
 
+      const {
+        body: legacyInitialLicense,
+        headers: legacyInitialLicenseHeaders,
+      } = await supertest.get('/api/xpack/v1/info').expect(200);
```

Comment:
temporarily adds tests for Legacy Licensing to have at least smoke-tests

Reply:
Should we move this to a separate legacy test so it can be more easily deleted?

Reply:
I wish we could. I need a comment in tests https://github.com/elastic/kibana/pull/51818#discussion_r352197629

Reply:
I'm going to add a client side tests in the followup PR. Probably I just create 2 separate FTR configs for NP & LP